plugins/image-source/obs-slideshow.c

```diff
@@ -208,6 +226,46 @@ static void add_file(struct slideshow *ss, struct darray *array,
 	*array = new_files.da;
 }
 
+static void clear_buffer(struct slideshow *ss, bool next)
+{
+	if (ss->paths.num < 5 || !ss->paths.num || !ss->files.num)
```

Comment:
I see this magic 5 used a lot. Perhaps there should be a constant for it instead?

Reply:
To clarify what Techjar is saying here, instead of using the number '5' throughout the code, you should create a macro at the top:
```C
#define MAX_LOADED 5
```
and then use that sort of macro throughout the codebase instead of using 5, both for the sake of code clarity and in case you want to change that number for whatever reason without having to change every instance where '5' is used (if it's used elsewhere).